projects/plugins/jetpack/_inc/client/recommendations/sidebar/mobile-app/index.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import React, { useCallback, useEffect } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import AppsBadge from './apps-badge';
+import { Layout } from '../layout';
+import Gridicon from 'components/gridicon';
+import { imagePath } from 'constants/urls';
+import analytics from 'lib/analytics';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const MobileApp = () => {
+	const features = [
+		__( 'Refined post and page editor', 'jetpack' ),
+		__( 'Manage multiple sites from one dash', 'jetpack' ),
```

Comment:
Should it be "dashboard"?

```suggestion
		__( 'Manage multiple sites from one dashboard', 'jetpack' ),
```